frontend/src/components/workspaces/DashboardLogic/DashboardLogic.tsx

```diff
@@ -15,16 +15,23 @@ function DashboardLogic() {
   //startPolling(20000);
 
   return !loading && data && !error ? (
-    <Dashboard
-      workspaces={
-        data.tenant?.spec?.workspaces?.map(workspace => {
-          return {
-            workspaceId: workspace?.workspaceRef?.workspaceId as string,
-            role: workspace?.role!,
-          };
-        }) ?? []
-      }
-    />
+    <>
+      <Dashboard
+        workspaces={
+          data.tenant?.spec?.workspaces?.map(workspace => {
+            return {
+              workspaceId: workspace?.workspaceRef?.workspaceWrapper
+                ?.itPolitoCrownlabsV1alpha1Workspace?.spec
+                ?.workspaceName as string,
+              role: workspace?.role!,
```

Comment:
Why use the `?` and the `!` to access the role

Reply:
The `?` is because the value returned by the query could be null, the `!` is because we trust that the value won't be null.

Reply:
Oh i thought that the second one would make the first one useless, sorry